src/store/wallet/effects/send/send.ts

```diff
@@ -445,11 +445,12 @@ export const buildTxDetails =
     }
 
     amount = Number(amount); // Support BN (use number instead string only for view)
-    const effectiveRate =
-      (invoice &&
-        invoiceCurrency &&
-        dispatch(getInvoiceEffectiveRate(invoice, invoiceCurrency, chain))) ||
-      undefined;
+    let effectiveRate;
+    if (invoice && invoiceCurrency && defaultAltCurrencyIsoCode === 'USD') {
```

Comment:
This change will skip the effective rate calculation for other currencies like EUR, GBP, etc. Instead we can do: `defaultAltCurrencyIsoCode === invoice.currency` and this PR will then fix the crypto pricing currency issue while keeping the effective rate properly calculating for other fiat currencies

Reply:
Thanks for reviewing. Using only `defaultAltCurrencyIsoCode === invoice.currency` will skip the effective rate calculation if for example `btc` is the currency selected when creating the invoice, however we could calculate the effective rate in that case because we have `usdAmount` in the invoice properties. Tell me if the changes make sense to you.